README.md

```diff
@@ -17,11 +17,14 @@ You need [Python3](https://www.python.org/downloads/) installed (and on the PATH
 
 You'll also need git installed and on the PATH, which you presumably should already if you have a git repository.
 
+Lastly, you'll need tkinter installed. You can install this with your package manager on Linux, or the installation exe on Windows. 
```

Comment:
I had a note like this before, but apparently tkinter is shipped with Python on every major platform (Win/Mac/Linux) so it's probably safe to assume no extra install is necessary.

Reply:
Ah maybe its not included with Debian or I unintentionally uninstalled it at some point cause I had to install it myself, but I removed that part in the most recent commit!